tests/pipelines/eks/awscli-cl2-load-with-addons-slos.yaml

```diff
@@ -61,6 +63,29 @@ spec:
   - default: https://raw.githubusercontent.com/awslabs/kubernetes-iteration-toolkit/main/tests/assets/eks_node_role.json
     name: node-role-cfn-url
     type: string
+  - name: namespace-prefix
+    default: "default"
+    description: "The prefix of namespaces for EKS Pod Identity test."
+  - name: namespace-count
+    default: "1"
+    description: "The number of namespaces for EKS Pod Identity test."
+  - name: pia-trust-policy-url
+    default: "https://raw.githubusercontent.com/awslabs/kubernetes-iteration-toolkit/main/tests/assets/eks-pod-identity/pia-trust-policy.json"
+    type: string
+  - name: pia-test-config-url
+    default: "https://raw.githubusercontent.com/awslabs/kubernetes-iteration-toolkit/main/tests/assets/eks-pod-identity/eks-pod-identity/config.yaml"
+  - name: pia-test-pod-spec-url
+    default: "https://raw.githubusercontent.com/awslabs/kubernetes-iteration-toolkit/main/tests/assets/eks-pod-identity/eks-pod-identity/pod-default.yaml"
+  - name: cl2-eks-pod-identity-pods
+    default: "5000"
+  - name: cl2-default-qps
+    default: "500"
+  - name: cl2-default-burst
+    default: "1000"
```

Comment:
lets not use this as default :D , it way above normal values.

Reply:
just changed the values